Flutter/lib/JoinSettle.dart

```diff
@@ -1,12 +1,9 @@
-import 'package:flutter/material.dart';
-import 'package:Settle/LobbyScreen.dart';
-import 'Server.dart';
-import 'Settle.dart';
-import 'app_localizations.dart';
+import 'import_all.dart';
 
 class JoinSettle extends StatefulWidget {
   final String hostName;
-  JoinSettle(this.hostName);
+  final DarkThemeProvider themeChange;
+  JoinSettle(this.hostName, this.themeChange);
```

Comment:
Again, it seems more appropriate for DarkThemeProvider to be accessed statically since there should only be 1 instance of it

Reply:
Resolved in commit `app-style 2c993e9`